.github/profile/README.md

```diff
@@ -0,0 +1,20 @@
+![az digital 2](https://github.com/az-digital/.github/assets/az-digital-2.png)
+
+
+# Welcome to Arizona Digital! 🌵
+A team of web-focused volunteers known as Arizona Digital meets weekly to build and test products like Arizona Bootstrap and Arizona Quickstart. Contributions are what keep Arizona Bootstrap moving and improving, and are in everyone's best interest.
+
+## Contribution 🌈
+As an open-source project, we count on the support and collaboration of amazing developers like you to keep Arizona Quickstart and Arizona Bootstrap moving forward. By contributing, you'll be helping to enhance the tools used by your fellow developers and making a real difference in improving the web experience across our campus.
+
+### Ways to contribute
+
+- Request an invite to the Friday Arizona Digital meetings by subscribing to the [UA Digital listserv](https://list.arizona.edu/sympa/info/ua-digital)
```

Comment:
I'm thinking this list should match the one on https://digital.arizona.edu (don't think we want to reference the listserv anymore):

```suggestion
- Drop in to one of the Arizona Digital [Friday meetings on Zoom](https://arizona.zoom.us/my/azdigital) every week from 10am to Noon.
```